src/utils/wallets.ts

```diff
@@ -0,0 +1,13 @@
+import { WalletId, StoreAccount, TAddress } from '@types';
+import { isSameAddress, isWeb3Wallet } from '@utils';
+
+export const isViewOnlyWallet = (walletId: WalletId): boolean => walletId === WalletId.VIEW_ONLY;
+
+export const isSenderAccountPresent = (
```

Comment:
Can we rename it to make it clear that it isn't just detecting an account but a specific type? Maybe with a comment too? Can we reflect it in the prop name?

Reply:
I just called it main type for now :-) We can look into using toggles on the different wallet types, but i think that will involve a fair bit of refactoring and doesn't need to be done now
![image](https://user-images.githubusercontent.com/29407814/91584636-c78a6480-e907-11ea-8711-4a234db53ee9.png)